benchmark/benchmarks.dart

```diff
@@ -52,13 +52,13 @@ class InsertBenchmark extends RTreeBenchmarkBase {
 class RemoveBenchmark extends RTreeBenchmarkBase {
   RemoveBenchmark(ScoreCollector collector) : super("Remove 5k", collector);
 
-  RTree<String> tree;
+  late RTree<String> tree;
   List<List<RTreeDatum>> items = [];
```

Comment:
```suggestion
  final items = <List<RTreeDatum<String>>>[];
```
Then you don't have to cast on line 61